Formula/keptn.rb

```diff
@@ -34,8 +34,6 @@ def install
     system bin/"keptn", "set", "config", "kubeContextCheck", "false"
 
     assert_match "Keptn CLI version: #{version}", shell_output(bin/"keptn version 2>&1")
-
-    assert_match "This command requires to be authenticated. See \"keptn auth\" for details",
-      shell_output(bin/"keptn status 2>&1", 1)
+    assert_match(/^Error: .*$/, shell_output(bin/"keptn status 2>&1", 1))
```

Comment:
Why can't we match a specific message anymore?

Reply:
We got different error outputs in the CI because it uses different os keystores. So that the macOS way passes and Linux does not.

Reply:
So let's do
```suggestion
    if OS.linux?
      assert_match(/^Error: .*$/, shell_output(bin/"keptn status 2>&1", 1))
    end
    if OS.mac?
      assert_match(/^Error: .*$/, shell_output(bin/"keptn status 2>&1", 1))
    end
```